test/src/runner/testRunnerScreenshotsOutput.js

```diff
@@ -235,8 +236,8 @@ describe('testRunnerScreenshotsOutput', function() {
 });
 
 function readDirPromise(dirName) {
-  return new Promise(function(resolve, reject) {
-    fs.readdir(dirName, function(err, result) {
+  return new Promise(function (resolve, reject) {
```

Comment:
these spaces shouldn't be added. also please remove all automatic lint fixes.

Reply:
done.